terminado/management.py

```diff
@@ -225,7 +225,7 @@ def kill_all(self):
         yield super(SingleTermManager, self).kill_all()
         self.terminal = None
 
-def MaxTerminalsReached(Exception):
```

Comment:
Ooops... I'm surprised that a class definition was a valid function definition, but now I think about it, there's nothing to make it invalid.